ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,898 @@
+/* *
+ *
+ *  Highcharts Breadcrumbs module
+ *
+ *  Authors: Grzegorz Blachliński, Karol Kołodziej
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import type {
+    AlignValue,
+    VerticalAlignValue
+} from '../Core/Renderer/AlignObject';
+import type {
+    CSSObject
+} from '../Core/Renderer/CSSObject';
+import type SVGElement from '../Core/Renderer/SVG/SVGElement';
+import Chart from '../Core/Chart/Chart.js';
+import H from '../Core/Globals.js';
+import O from '../Core/Options.js';
+const {
+    defaultOptions
+} = O;
+import U from '../Core/Utilities.js';
+const {
+    addEvent,
+    defined,
+    extend,
+    format,
+    merge
+} = U;
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        interface BreadcrumbsOptions {
+            buttonPadding?: number;
+            enabled?: boolean;
+            events?: BreadcrumbsButtonsEventsOptions;
+            floating?: boolean;
+            format?: string;
+            formatter?: BreadcrumbsButtonsFormatter;
+            position?: BreadcrumbsAlignOptions;
+            separator?: string;
+            showOnlyLast?: boolean;
+            style?: CSSObject;
+            useHTML?: boolean;
+            zIndex?: number;
+        }
+        interface BreadcrumbsAlignOptions {
+            align?: AlignValue;
+            verticalAlign?: VerticalAlignValue;
+            alignByTranslate?: boolean;
+            x: number;
+            y: number;
+        }
+        interface BreadcrumbsClickCallbackFunction {
+            (e: Event, breadcrumbs: Breadcrumbs): (boolean|undefined);
+        }
+        interface BreadcrumbsButtonsEventsOptions {
+            click?: BreadcrumbsClickCallbackFunction;
+        }
+        interface BreadcrumbsButtonsFormatter {
+            (
+                breadcrumb: Array<any>,
```

Comment:
Could we define what's passed on? To avoid using `any`

Reply:
We should not use casting to `any`.

You can create your own type and use it here.